tools/run_tests/performance/scenario_config.py

```diff
@@ -427,7 +454,7 @@ def scenarios(self):
           'server_type': 'SYNC_SERVER',
           'security_params': secargs,
           'core_limit': 0,
-          'async_server_threads': 1,
+          'async_server_threads': 0,
```

Comment:
I would definitely recommend core_limit 1 and async_server_threads 1 in this case since the server is C++ and these make sense.